<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIClass</class>
 <widget class="QMainWindow" name="GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple image editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gui.qrc">
    <normaloff>:/GUI/Resources/mainicon.png</normaloff>:/GUI/Resources/mainicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gbImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QPushButton" name="btnOpen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Открыть</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSave">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSaveAs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить как...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbFilters">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>151</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>Фильтры</string>
    </property>
    <widget class="QPushButton" name="btnGrayWorld">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&quot;Серый мир&quot;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGrayscale">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Grayscale</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHistEq">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Автоконтраст</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStretchChan">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Контраст</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSobel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Собель</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMedian">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Медиана</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGauss">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Фильтр Гаусса</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLinear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Произвольный ЛФ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnWaves">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Волны</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMyFilt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Мой фильтр</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="gvMain">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>571</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbMain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>731</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Geometry">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>151</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Преобразования геометрии</string>
    </property>
    <widget class="QPushButton" name="btnRotate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Поворот</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnResize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Масштабирование</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="mnuOpen"/>
    <addaction name="mnuSave"/>
    <addaction name="mnuSaveAs"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Фильтры</string>
    </property>
    <addaction name="mnuGrayscale"/>
    <addaction name="mnuGrayWorld"/>
    <addaction name="mnuHistEq"/>
    <addaction name="mnuStretchChan"/>
    <addaction name="mnuSobel"/>
    <addaction name="mnuMedian"/>
    <addaction name="mnuGauss"/>
    <addaction name="mnuLinear"/>
    <addaction name="mnuWaves"/>
    <addaction name="mnuMyFilt"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="about"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Геометрия</string>
    </property>
    <addaction name="mnuRotate"/>
    <addaction name="mnuResize"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="mnuOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="mnuSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="mnuSaveAs">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="mnuGrayWorld">
   <property name="text">
    <string>&quot;Серый мир&quot;</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Медианная фильтрация</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="mnuGrayscale">
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="mnuHistEq">
   <property name="text">
    <string>Автоконтраст</string>
   </property>
  </action>
  <action name="mnuStretchChan">
   <property name="text">
    <string>Контраст</string>
   </property>
  </action>
  <action name="mnuSobel">
   <property name="text">
    <string>Собель</string>
   </property>
  </action>
  <action name="mnuMedian">
   <property name="text">
    <string>Медиана</string>
   </property>
  </action>
  <action name="mnuGauss">
   <property name="text">
    <string>Фильтр Гаусса</string>
   </property>
  </action>
  <action name="mnuRotate">
   <property name="text">
    <string>Поворот</string>
   </property>
  </action>
  <action name="mnuResize">
   <property name="text">
    <string>Масштабирование</string>
   </property>
  </action>
  <action name="mnuLinear">
   <property name="text">
    <string>Произвольный ЛФ</string>
   </property>
  </action>
  <action name="mnuWaves">
   <property name="text">
    <string>Волны</string>
   </property>
  </action>
  <action name="NeonGlow">
   <property name="text">
    <string>Неоновый свет</string>
   </property>
  </action>
  <action name="mnuMyFilt">
   <property name="text">
    <string>Мой фильтр</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gui.qrc"/>
 </resources>
 <connections/>
</ui>
